### PR TITLE
Add navbar(non-functional atm), delete other paragraphs and image to …

### PR DESCRIPTION
**Pull request by ethanimooney**
…make room to update. Currently adding the main index.html functionality, more commits to come

### CODE REVIEW

**Comment by ethanimooney** [comment]
Sorry about the large-ish commit there, got carried away and forgot to commit earlier. 

Adds flexbox functionality to entire site, adds about me section with pictures, bio, and skills/projects section. Also adds connect buttons, links to 4 social media.

I am sure some stuff violates the style guides, I plan on fixing that in another PR, but feel free to point out violations as you see them.

File: portfolio/pom.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@
   <version>1</version>
   <packaging>war</packaging>
 
+
```
Comment (shaquille-hall) [inline portfolio/pom.xml:11]: Perhaps you can revert this file. There doesn't seem to be anything changing.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -3,18 +3,79 @@
   <head>
     <meta charset="UTF-8">
     <title>Ethan's Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/75dadd3f2b.js" crossorigin="anonymous"></script>
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/index.html:9]: HTML loads documents from top to bottom. Because of this, it's a common practice to add script tags just above your closing body tag. This avoids page render delays due to expensive js.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -3,18 +3,79 @@
...
+                        My main hobbies include photography, gaming, technology repair, and bouldering.</br>
+                        Channeling my inner nerd, I also co-lead a Pokemon TCG league at a local card shop.
+                        </br>
+                        </br>
+                        I began programming in 7th grade, when I was introduced to Scratch. I really enjoyed
+                        it, so once I got to my sophomore year of high school, I had the chance to enroll
+                        in a local vocational technology center's Computer Science Academy. This was where
+                        my love of coding really took hold and I learned enough to make me decide to pursue
+                        computer science as a degree and a career.
+                    </p>
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/index.html:51]: The line length will expand for as much as the browser window allows. This will particularly be the case for the last paragraph. If you'd like, one solution may be to wrap the text using css.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -3,18 +3,79 @@
...
+
+                  <h3>Projects</h3>
+                  <p>wheell [webapp that gives wheelchair users detailed information about searched locations]
+                      </br></br>
+                      udining.com [re-designed website for a local university dining company]
+                      </br></br>
+                      swablu [discord server management bot]
+                      </br></br>
+                      ethanmooney.me [personal website]
+                  </p>
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/index.html:68]: Would a list be better here?

File: portfolio/pom.xml (hunk 1 above)
Comment (ethanimooney) [reply portfolio/pom.xml:11]: Got it. Added something to make re-running the project easier and it didn't work so I deleted it, must have left a few spaces or something.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (ethanimooney) [reply portfolio/src/main/webapp/index.html:9]: Ah okay. The walk through had it up there but I'll move it down.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (ethanimooney) [reply portfolio/src/main/webapp/index.html:51]: I think it is wrapping most of it. I only have the few </br> at the end of some lines because I wanted them on a new line. the others are just for spacing.

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (ethanimooney) [reply portfolio/src/main/webapp/index.html:68]: Yes it would -- changing

**Comment by ethanimooney** [comment]
Alright, I committed again. 

TL;DR: Addresses all PR comments, ensures the following of Google style guidelines, updates the first minor piece of work in the week 3 curriculum.

I went in and fixed the things you suggested, converted my CSS and HTML files to properly follow the Google styles guides (if only just to get in the habit of following them). Also tried to revert that pom.xml file back to original, removed an extra line.

Also started on the Week #3 curriculum so DataServlet.java was updated, but a very minor change..

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (shaquille-hall) [reply portfolio/src/main/webapp/index.html:49]: Ok. I see the corresponding css.

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (shaquille-hall) [reply portfolio/src/main/webapp/index.html]: Did you push the change? I'm not seeing it on my end.

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -25,12 +23,12 @@ <h1 class="header-item">Ethan Mooney</h1>
...
-                     <img src="/images/ethan-p3.jpeg" width="100px">
-                     <img src="/images/ethan-p4.jpeg" width="100px">
-                     <img src="/images/ethan-p5.jpeg" width="100px">
-                     <img src="/images/ethan-p6.JPG" width="100px">
+                     <img src="/images/ethan-p1.jpeg" alt="Ethan on bridge" width="100px">
+                     <img src="/images/ethan-p2.jpeg" alt="Ethan on cliff side" width="100px">
+                     <img src="/images/ethan-p3.jpeg" alt="Ethan by Google sign" width="100px">
+                     <img src="/images/ethan-p4.jpeg" alt="Ethan and CSSI friends" width="100px">
+                     <img src="/images/ethan-p5.jpeg" alt="Ethan in wheat field" width="100px">
+                     <img src="/images/ethan-p6.JPG" alt="Ethan CSSI graduation" width="100px">
```
Comment (shaquille-hall) [inline portfolio/src/main/webapp/index.html:31]: nit: This file extension is inconsistent with the others.

Comment (ethanimooney) [reply portfolio/src/main/webapp/index.html:31]: Yeah idk why it did that, all came from the same place, can change.
